stack.yaml

```diff
@@ -1,5 +1,6 @@
-resolver: lts-17.2
+resolver: lts-18.19
 flags:
   slack-web:
     cli: true
-extra-deps: []
+extra-deps:
+- mintty-0.1.2
```

Comment:
Is this used somewhere?

Reply:
This was necessary to avoid https://github.com/commercialhaskell/stackage/issues/6381, but now it can be unnecessary anymore. Let me get it back.